### DISCUSSION TITLE
[FR] Allow date field in table field to show both date and time

### DISCUSSION THREAD

**Discussion by davist11** [craftcms/cms#9041] [Ideas]
### Description

When adding a date field in a table field, you cannot add a single field for both date and time like the standalone date field.

![image](https://user-images.githubusercontent.com/199985/119029534-bc380700-b976-11eb-9b56-0cceaa7c8377.png)

![image](https://user-images.githubusercontent.com/199985/119029576-c6f29c00-b976-11eb-8041-a2ec30b0b5a7.png)

Would be really cool to utilize the functionality for how you add dropdown options for handling this setting for in-table date fields

![image](https://user-images.githubusercontent.com/199985/119029629-df62b680-b976-11eb-9413-caa1fca75e8b.png)

**Comment by brandonkelly** [comment]
Related: #7512
